src/Integration.Vsix/CFamily/CFamilyAnalyzerOptions.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SonarLint.VisualStudio.Core;
```

Comment:
This is now redundant - I moved this class to `Core` at the same time as `IAnalyzerOptions`.